extensions/cloudflare-email-routing/package.json

```
@@ -0,0 +1,85 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "cloudflare-email-routing",
+  "title": "Cloudflare Email Routing",
+  "description": "Create, list, edit, and delete Cloudflare email aliases directly from Raycast.",
+  "icon": "extension-icon.png",
+  "author": "erayack",
+  "categories": [
+    "Productivity",
+    "Communication"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "list-aliases",
+      "title": "List Email Aliases",
+      "subtitle": "List and manage all your Cloudflare email aliases",
```

```suggestion
      "subtitle": "Cloudflare Email Routing",
```
The subtitle is always the same as title of the extension unless it is an exception.